rust/repositories.bzl

```diff
@@ -40,11 +40,11 @@ load_arbitrary_tool = _load_arbitrary_tool
 
 # Note: Code in `.github/workflows/crate_universe.yaml` looks for this line, if you remove it or change its format, you will also need to update that code.
 DEFAULT_TOOLCHAIN_TRIPLES = {
-    "aarch64-apple-darwin": "rust_darwin_aarch64",
+    "aarch64-apple-darwin": "rust_macos_aarch64",
```

Comment:
I *believe* this is technically a backwards incompatible change because the `@@rust_darwin_aarch64__aarch64-apple-darwin__stable_tools` repository will no longer exist

Reply:
This could potentially be backward incompatible but I don't think folks explicitly rely on these repository names so I'm willing to accept the breakage 😅